doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -48,7 +48,11 @@ This specification is licensed under the
 
 ## Conformance
 
-All the interfaces described by this specification are mandatory and must be implemented in order to be compliant with. 
+This specification describes the "full" compliance level.
+
+However, next iterations of this specification could define optionality
+in some parts and the compliance level (or roles/profiles)
+corresponding to them.
```

Comment:
It depends. I agree that *ungoverned* optionality could be a interoperability hole, but it shouldn't be the case if a mechanism to express optional capabilities is defined (as the one discussed at https://github.com/telefonicaid/fiware-orion/issues/2110). And, in that case, we will gain the advantages of optionality: lower the implementation barrier (it is easier to implement part of the API than the whole API).

For TID optionality is not important, as our aim is to implement the whole API. But I understand that for some partners (NEC, Orange, etc.) it is.

Such mechanism to express optional capabilities is yet under discussion and will be part of next stable releases of the API. The purpose of this paragrah is to express that ciscustance. However, if you find more appropiate to include in a side-section (the one for Release Notes), it is ok with me.

@tobiasjacobs , @marcc-orange , what do you think?